WcaOnRails/app/views/country_bands/edit.html.erb

```diff
@@ -0,0 +1,38 @@
+<div class="container">
+  <%= link_to("Back to country bands", country_bands_path) %>
+  <h1 class="text-center">Editing Country Band <%= @number %></h1>
+  <%= simple_form_for :country_band, url: country_band_path(id: @number), method: :put do |f| %>
+    <%= f.input :number, as: :hidden %>
+    <%= simple_fields_for :countries do |c| %>
+      <%= c.input :iso2s, as: :string, label: false, input_html: { id: "iso2s" } %>
+    <% end %>
+    <%= f.button :submit, class: "btn-primary", value: "Save the country band" %>
+  <% end %>
+</div>
+<script>
+  const optionElement = '<div><%= flag_icon "%iso2" %> %name</div>';
+  const itemElement = '<div class="country"><div class="flag"><%= flag_icon "%iso2" %></div><div class="name">%name</div></div>';
```

Comment:
Hate to say that, but as we don't transpile inline JS, it'd be better to use `var` here as well.